### DISCUSSION TITLE
part-ii/2016-64/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#322] [Question comments]
Part II 2016 Galois Theory: Paper 2, Section II, H 

https://questions.tripos.org/part-ii/2016-64/

**Comment by DannyDuberstein** [comment]
(a) PET, take prim elt's min poly over F, only finite options. And coefficients of the min poly as [L:K(coeffs)] = [L:F] (as came up in 2018, except this question demands that you see this all in advance)
(b)No, have non-real roots of its minpoly.
